src/java.base/share/classes/java/lang/StringConcatHelper.java

```diff
@@ -133,7 +133,7 @@ static long mix(long lengthCoder, String value) {
      *                    the upper word.
      * @param value       value to mix in
      * @return            new length and coder
-     * @since 21
+     * @since 22
```

Comment:
> Wasn't sure about that. Thx.

When in doubt, JEP 12.  Alex provided good guidance for API authors on how `@since` should be used with preview APIs.